### PR TITLE
papiNet API 1.3.0 is ready ;-) | Feature/availability

### PR DESCRIPTION
**Pull request by patricekrakow**
Adding the Availability use case.

### CODE REVIEW

**Review by MrNordenberg — CHANGES_REQUESTED** [review]
This is a MAJOR not backward compatible change, since an implemented API would need to be changed due to that quite a few properties now are required, e.g. "href" in PaginationLinks first, prev and next, further the rows 1270-1274.

Hence the version should be papiNet API 2.0.0 in accordance with the semantic versioning MAJOR.MINOR.PATCH. Study the summary of semantic versioning presented at semver.org.

**Review by globaltopmedia — CHANGES_REQUESTED** [review]
Some small stuff,  plus the major version discussion.

**Comment by larsolofsson** [comment]
Please review and resolve 7 issues, #49 to #55

**Review by larsolofsson — CHANGES_REQUESTED** [review]
Please review and resolve 7 issues, https://github.com/papinet/papiNet-API/issues/49 to https://github.com/papinet/papiNet-API/issues/55

File: 1.3.0/mock/upload.cmd [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cmd /c gcloud compute ssh elfralalo@node-01 --command="mkdir /home/elfralalo/papinet-mock-service-1-2-0"
```
[the remark this answers is not in the document]
Comment (patricekrakow) [reply 1.3.0/mock/upload.cmd:1]: Indeed. I should use a "USER" variable in that script.

**Comment by larsolofsson** [comment]
**format: uuid** is missing for property **id** in GetLocationByID:

**Pull request merged**